mbuild/formats/cassandramcf.py

```diff
@@ -120,22 +120,28 @@ def write_mcf(
                 )
     if lj14 is None:
         if len(structure.adjusts) > 0:
-            type1_eps = structure.adjusts[0].atom1.epsilon
-            type2_eps = structure.adjusts[0].atom2.epsilon
-            scaled_eps = structure.adjusts[0].type.epsilon
             if (
                 structure.combining_rule == "geometric"
                 or structure.combining_rule == "lorentz"
             ):
-                combined_eps = sqrt(type1_eps * type2_eps)
-                if combined_eps == 0:
+                combined_eps_list = [
```

Comment:
This is just for my own understanding, but is the difference between the "geometric" and "lorentz" combining rule only come into play with calculating the combined sigma values in the lennard jones equation? Combined `epsilon` values are done with the "geometric" method regardless?

Reply:
Yes

Reply:
There might be other mixing rules, but for geometric and lorentz, geometric mixing rules are applied to epsilon.

https://manual.gromacs.org/documentation/current/reference-manual/functions/nonbonded-interactions.html#equation-eqnlorentzberthelot